cmd/microcloud/ask.go

```diff
@@ -148,8 +148,7 @@ func (c *CmdControl) askAddress(autoSetup bool, listenAddr string) (string, *net
 	return listenAddr, iface, subnet, nil
 }
 
-func (c *CmdControl) askDisks(sh *service.Handler, systems map[string]InitSystem, autoSetup bool, wipeAllDisks bool) error {
-	_, bootstrap := systems[sh.Name]
+func (c *CmdControl) askDisks(sh *service.Handler, systems map[string]InitSystem, autoSetup bool, wipeAllDisks bool, bootstrap bool) error {
```

Comment:
This is totally a nit, but the sum of these two/three (`autoSetup` and `bootstrap` (and `wipeAllDisks`)) seems like it's being passed around a lot here. It would be more readable at the call site if this were factored into some kind of `AskFlags` structure.

Reply:
Yeah, that's true. A lot of this config (even the systems map) should probably be part of the receiver of these methods.

Reply:
I've added 2 commits here (though it might not be the best place for it) to add a new `initConfig` struct. 

This holds all the arguments that are passed to every helper, so that they can all be receivers of that type and access the fields.

Reply:
Thanks, this looks better.